### PR TITLE
Fix private tangle action

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: .github/actions/private-tangle/setup/action.yml [hunk 1]
```diff
@@ -18,13 +18,13 @@ runs:
 
         # remove the last 5 lines and add docker compose with `-d` to run it in the background
         sed -i -n -e :a -e '1,5!{P;N;D;};N;ba' run.sh
-        echo "docker compose -f \$DOCKER_COMPOSE_FILE up -d" >> run.sh
+        echo "docker compose -f docker-compose.yml up -d" >> run.sh
```
Comment (thibault-martinez) [inline .github/actions/private-tangle/setup/action.yml:21]: Looks like this env var is never defined, not sure where this is supposed to happen

Comment (Thoralf-M) [reply .github/actions/private-tangle/setup/action.yml:21]: In the run.sh file before https://github.com/iotaledger/iota-core/blob/5851ebd9b1fe272be6a0f86e92f0841f457bbfd5/tools/docker-network/run.sh#L17

Comment (Thoralf-M) [reply .github/actions/private-tangle/setup/action.yml:21]: Not sure why it doesn't like it anymore, but just directly using `docker-compose.yml` should be fine

**Review by Thoralf-M — APPROVED** [review]

File: .github/actions/private-tangle/setup/action.yml (hunk 1 above)
Comment (thibault-martinez) [reply .github/actions/private-tangle/setup/action.yml:21]: But the env var is evaluated in the shell before being written in run.sh, not sure how that ever worked...

Comment (Thoralf-M) [reply .github/actions/private-tangle/setup/action.yml:21]: With the `\` it shouldn't get evaluated

Comment (thibault-martinez) [reply .github/actions/private-tangle/setup/action.yml:21]: Should we just remove the file name? They don't have it in the original docker compose up

Comment (thibault-martinez) [reply .github/actions/private-tangle/setup/action.yml:21]: Ahhhh but they have removed `DOCKER_COMPOSE_FILE` from run.sh I see

**Review by DaughterOfMars — APPROVED** [review]

**Review by tadzik — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**